plugins/userdata/JetUserData.cc

```diff
@@ -435,8 +449,30 @@ void JetUserData::fillJEC(pat::Jet& jet, const edm::Event& iEvent, const edm::Ev
 	jet.addUserFloat("L1OffJEC", jet.correctedJet("L1FastJet").pt() / jet.correctedJet("Uncorrected").pt());
 }
 
-void JetUserData::fillJER(pat::Jet& jet) const {
+void JetUserData::fillJER(pat::Jet& jet, const edm::Event& iEvent, 
+		JME::JetResolution resolution, JME::JetResolutionScaleFactor resolution_sf) const {
```

Comment:
If `resolution` and `resolution_sf` are made member variables, this function will look more simple.